<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QMainWindow" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>591</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SecondWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="num1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="num2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labela">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Операнд 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelb">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Операнд 2</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Операции</string>
    </property>
    <widget class="QRadioButton" name="op_min">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="op_slosh">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="op_umn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="op_del">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="end">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="finaltext">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelresult">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelerror">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>43</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
